server/controller/business.js

```diff
@@ -0,0 +1,29 @@
+import businesses from '../model/business';
```

Comment:
It will be great if you destructure the values you get from req.body as seen in the object in `business.push`. This way, you can have cleaner code without recurring `req.body`. I think adding status to your res.json will also be great, that way we get to know if the operation is successful or not.

Great work.

Reply:
Okay samson, I appreciate. I am actually going to look that up now and change my implementation.